### PR TITLE
 Change bucket type to table in error messages. [JIRA: RIAK-2347]

### PR DESCRIPTION
**Pull request by andytill**
``` erlang
56> riakc_ts:query(Pid, "DESCRIBE DERP").       
{error,{1007,<<"Time Series table DERP is missing.">>}}
```

### CODE REVIEW

File: src/riak_kv_pb_timeseries.erl [hunk 1]
```diff
@@ -487,24 +487,24 @@ missing_helper_module(Table, BucketProps) when is_binary(Table), is_list(BucketP
...
 
 %%
--spec missing_type_response(BucketType::binary()) -> #rpberrorresp{}.
-missing_type_response(BucketType) ->
+-spec missing_type_response(Table::binary()) -> #rpberrorresp{}.
+missing_type_response(Table) ->
     make_rpberrresp(
         ?E_MISSING_TYPE,
-        flat_format("Bucket type ~s is missing.", [BucketType])).
+        flat_format("Time Series table ~s is missing.", [Table])).
```
Comment (alexmoore) [inline src/riak_kv_pb_timeseries.erl]: I'd say `s/is missing/does not exist/`  "is missing" sounds like it was there, but now it's not.

**Comment by alexmoore** [comment]
+1 7edba0b

**Comment by DSomogyi** [comment]
create jira issue

**Comment by Basho-JIRA** [comment]
https://github.com/basho/riak_kv/pull/1322 

**_[posted via JIRA by Derek Somogyi]**_

**Comment by andytill** [comment]
@borshop merge

**Pull request merged**